docs/mint.json

```diff
@@ -83,6 +83,7 @@
             "medical-api/api-reference/patient/create-patient",
             "medical-api/api-reference/patient/get-patient",
             "medical-api/api-reference/patient/get-patient-external-id",
+            "medical-api/api-reference/patient/create-patient-external-sync",
```

Comment:
I'd place this after `Delete Patient` - I think the main CRUD operations should be close together to help with understanding it.